tool_commands/guess_indicators/test_guess_indicators.py

```diff
@@ -0,0 +1,238 @@
+    all manholes (manhole_outlet + manhole_pump + other manholes)
+    :return: a list with tuples (manhole.id, manhole.manhole_indicator)  """
+    # only 1 of kwargs may be True
+    assert sum([outlet_bool, pump_bool, all_bool]) == 1
+    if pump_bool:
+        ids = session.query(Pumpstation.connection_node_start_id)
+    elif outlet_bool:
+        ids = session.query(BoundaryCondition1D.connection_node_id)
+    if pump_bool or outlet_bool:
+        manholes_id_indicator = get_pump_or_outlet_manholes(session, ids)
+    if all_bool:
+        manholes_id_indicator = get_all_manholes(session)
+    return manholes_id_indicator
+
+
+def get_all_pipes(session):
+    pipes = {}
+    # only return manhole
+    sql_pipes = session.query(Pipe)
+    for pipe in sql_pipes:
+        # fill dict
+        pipes[pipe.id] = (pipe.friction_type, pipe.friction_value, pipe.material)
+    return pipes
+
+
+# def get_all_pipes_friction_type(session):
+#     pipes_id_friction_type = {}
+#     # only return manhole
+#     sql_pipes = session.query(Pipe)
+#     for pipe in sql_pipes:
+#         # fill dict
+#         pipes_id_friction_type[pipe.id] = pipe.friction_type
+#     return pipes_id_friction_type
+#
+#
+# def get_all_pipes_friction_value(session):
+#     pipes_id_friction_value = {}
+#     # only return manhole
+#     sql_pipes = session.query(Pipe)
+#     for pipe in sql_pipes:
+#         # fill dict
+#         pipes_id_friction_value[pipe.id] = pipe.friction_value
+#     return pipes_id_friction_value
+
+
+def empty_manhole_indicator(session, manholes_pre_empty):
+    """ empty column manhole_indicator (set is to NULL) of table v2_manhole """
+    manholes_ids = list(manholes_pre_empty)
+    up = (
+        update(Manhole)
+        .where(Manhole.id.in_(manholes_ids))
+        .values(manhole_indicator=None)
+    )
+    session.execute(up)
+    session.commit()
+
+
+def empty_pipe_friction_value(session, pipes_pre_empty):
+    """ empty column manhole_indicator (set is to NULL) of table v2_manhole """
+    pipes_ids = list(pipes_pre_empty)
+    up = (
+        update(Pipe)
+        .where(Pipe.id.in_(pipes_ids))
+        .values(friction_value=None, friction_type=None)
+    )
+    session.execute(up)
+    session.commit()
+
+
+@pytest.fixture()
+def tmp_sqlite_path(tmpdir):
+    sqlite_filename = "v2_bergermeer.sqlite"
+    orig_sqlite_path = TEST_DATA_DIR.joinpath(
+        "testmodel", "v2_bergermeer", sqlite_filename
+    )
+    tmp_sqlite_dir = Path(tmpdir)
+    tmp_sqlite_path = tmp_sqlite_dir.joinpath(sqlite_filename)
+    shutil.copy2(orig_sqlite_path, tmp_sqlite_path)
+    return tmp_sqlite_path
+
+
+@pytest.fixture()
+def db(tmp_sqlite_path):
+    db_type = "spatialite"
+    db_set = {"db_path": tmp_sqlite_path}
+    db = ThreediDatabase(db_set, db_type)
+    return db
+
+
+@pytest.fixture()
+def guesser(db):
+    return Guesser(db)
+
+
+def test_guess_manhole_indicator(db, guesser):
```

Comment:
De guesser fixture maakt al van de db fixture gebruik. Hier grijp je ze allebei. Komt daar interferentie van? Of valt het mee?

Ik zat even naar de opstapeling van fixtures te kijken: volgens mij gebruik je altijd de combinatie db+guesser. Kan dat dan niet één fixture worden i.p.v. de 4 of 5 die er nu zijn?